admin/agentlocal/agentlocal.go

```diff
@@ -115,10 +120,6 @@ func GetStatus(requestNetworkInfo NetworkInfo) (*Status, error) {
 		return nil, ErrNotSetUp
 	}
 
-	if p.RunsOnNodeID == "" {
```

Comment:
when agent can't connect to server then `RunsOnNodeID` is empty. But some other information is still available like agent id, for example. 

And instead of returning error message in this case we can return status information, but with some empty fields. Here is an example of such case: 

```{
 "pmm_agent_status": {
   "agent_id": "/agent_id/d4adc705-d24d-4485-8174-7ca8792880d2",
   "node_id": "",
   "server_url": "https://127.0.0.1:443/",
   "server_insecure_tls": true,
   "server_version": "",
   "agent_version": "2.28.0-896-gaaa90bc2-dirty",
   "agents": [],
   "connection_uptime": "99.00", 
   "connected": false
 },
 "pmm_admin_version": "2.28.0-893-ge3484c6a-dirty"
}```

Reply:
Before we stopped execution here.
What happens in case agent is not connected and we run a command such as `pmm-admin list` which relies on `NodeID` not being empty?

Reply:
@michal-kralik in this case I can see that `pmm-admin list` command locally returns error `Post "https://127.0.0.1:443/v1/inventory/Services/List": dial tcp 127.0.0.1:443: connect: connection refused`, when agent lost connection to server.